docs/release/release-2.0.rst

```diff
@@ -0,0 +1,38 @@
+===========
+Release 2.0
+===========
+
+New Features
+------------
+* Make presto-admin log and configuration directories configurable. They can be
+  set using the environment variables ``PRESTO_ADMIN_LOG_DIR`` and
+  ``PRESTO_ADMIN_CONFIG_DIR``.
+* Change the default configuration directory to ``~/.prestoadmin`` and the
+  default log directory to ``~/.prestoadmin/log``.
+* Remove the requirement for running and installing presto-admin with sudo.
```

Comment:
I disagree with Anton, I think you should mention that you need sudo access on all the nodes. A lot of people (including Apple) are confused about this.